packages/components/src/components/hds/application-state/header.hbs

```diff
@@ -3,18 +3,24 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 <div class="hds-application-state__header" ...attributes>
+  {{#if @errorCode}}
+    <Hds::Text::Body
+      class="hds-application-state__error-code"
+      @tag="div"
+      @size="100"
+      @weight="medium"
+      @color="foreground-faint"
```

Comment:
This doesn't seem to be set (`.hds-application-state__header` color `strong` is being inherited)

Visual reference:
<img width="563" alt="image" src="https://github.com/user-attachments/assets/e39b6423-089b-4462-bc0f-1093b222da37">

```suggestion
      @color="faint"
```

Reply:
Nice catch!